masonry/src/render_root.rs

```diff
@@ -576,7 +576,11 @@ impl RenderRoot {
         // We request a redraw if either the render tree or the accessibility
         // tree needs to be rebuilt. Usually both happen at the same time.
         // A redraw will trigger a rebuild of the accessibility tree.
-        if self.root_state().needs_paint || self.root_state().needs_accessibility {
+        // TODO - We assume that a relayout will trigger a repaint
+        if self.root_state().needs_paint
+            || self.root_state().needs_accessibility
+            || self.root_state().needs_layout
```

Comment:
We also need to fix all of the widgets which do:
```rust
ctx.request_layout();
ctx.request_paint();
```

Reply:
Done.